docs/monitoring-metrics.md

```diff
@@ -30,10 +30,10 @@ Inventory of health metrics collected by the Jenkins OpenTelemetry integration:
 | ci.pipeline.run.started          | 1     |            |                   | Job started |
 | ci.pipeline.run.completed        | 1     |            |                   | Job completed |
 | ci.pipeline.run.aborted          | 1     |            |                   | Job aborted |
-| jenkins.queue.waiting            | 1     |            |                   | Number of waiting items in queue |
-| jenkins.queue.blocked            | 1     |            |                   | Number of blocked items in queue |
-| jenkins.queue.buildable          | 1     |            |                   | Number of buildable items in queue |
-| jenkins.queue.left               | 1     |            |                   | Total count of left items |
+| jenkins.queue.waiting            | 1     |            |                   | Number of waiting tasks in the queue that are not blocked. tasks with status `waiting` or `buildable` or `pending` (see `Queue#getUnblockedItems()`) |
```

Comment:
link to https://javadoc.jenkins.io/hudson/model/Queue.html#getUnblockedItems-- ?